.evergreen/perfcomp/cmd/perfcomp/compare.go

```diff
@@ -0,0 +1,131 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"math"
+	"os"
+	"sort"
+	"strings"
+	"text/tabwriter"
+	"time"
+
+	"github.com/mongodb-labs/drivers-evergreen-tools/perfcomp"
+	"github.com/spf13/cobra"
+)
+
+// For support for other projects, a performance context needs to be created and added here.
+var projectToPerfContext = map[string]string{
+	"mongo-go-driver": "GoDriver perf task",
```

Comment:
What are your thoughts on making this a required flag, instead of requiring implementers to extend the code?

```
perfcomp --context="GoDriver perf task"
```